packages/camera/camera/lib/src/camera_controller.dart

```diff
@@ -232,26 +232,33 @@ class CameraValue {
 /// To show the camera preview on the screen use a [CameraPreview] widget.
 class CameraController extends ValueNotifier<CameraValue> {
   /// Creates a new camera controller in an uninitialized state.
+  /// Deprecated, use [withSettings].
```

Comment:
nit: This needs a space after the first sentence. And add a please as a personal preference improvement

```suggestion
  ///
  /// Deprecated. Please use [withSettings].
```